www/%team/history/index.html.spt

```diff
@@ -0,0 +1,88 @@
+from datetime import datetime
+
+from aspen import Response
+
+from gratipay.utils import get_team
+from gratipay.utils.team_history import get_end_of_year_totals, iter_team_payday_events
+
+[-----------------------------------------------------------------------------]
+
+team = get_team(state)
+
+if user.ANON:
+    raise Response(401)
+if not (user.ADMIN or user.participant.username == team.owner):
+    raise Response(403)
+
+banner = team.name
+title = _("History")
+
+current_year = datetime.utcnow().year
+try:
+    year = int(request.qs.get('year', current_year))
+except ValueError:
+    raise Response(400, "Bad Year")
+years = list(range(current_year, team.ctime.year-1, -1))
+
+if year not in years:
```

Comment:
Idea here was to ensure that a person will not put a year in the URL and that year could be greater than the current year or less than the year the team/project 'ctime`